api-reference/docstubs/api/calendar-post-calendarpermissions.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Add calendarPermissions"
+description: "Add calendarPermissions by posting to the calendarPermissions collection."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# Add calendarPermissions
+
+Namespace: microsoft.graph
+
+Add calendarPermissions by posting to the calendarPermissions collection.
```

Comment:
I think this should be 
title: Create calendarPermissions
description:  Create a new calenderPermission object.